meta-lmp-base/recipes-support/fio-diag/fio-diag_0.1.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "foundries diagnostic tool"
```

Comment:
nitpicks:

I'd just extend the summary a bit, something like:
 `SUMMARY = "Foundries.io Diagnostic Tool for a Device"`. 

also good to have HOMEPAGE var defined, as it has a readme about the tool usage
`HOMEPAGE = "https://github.com/foundriesio/lmp-tools/tree/master/device-scripts"`

Reply:
both have been added